images/inventory-generator/Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM centos:7
+
+RUN INSTALL_PKGS="openssh-clients wget git" \
+    && yum install -y --setopt=tsflags=nodocs $INSTALL_PKGS \
+    && EPEL_PKGS="PyYAML ansible python2-boto" \
```

Comment:
EPEL packages sometimes shadow official packages, which we would usually prefer. There are two basic ways to prevent this - this is one - the other is to define the EPEL repo but leave it inactive and only enable it for the yum transactions needed. Either way you end up separating out the EPEL dependencies.

The other reason to do this is because it identifies which packages are "gaps" in the official packages. In the downstream these tend to require some different treatment, either shipping packages in another channel, getting them via RHSCL, or making them unnecessary somehow. And if things change and the packages are no longer needed... it becomes obvious you can get rid of EPEL.